core/src/main/scala/org/apache/spark/errors/SparkCoreErrors.scala

```diff
@@ -30,7 +30,7 @@ import org.apache.spark.storage.{BlockId, BlockManagerId, BlockNotFoundException
 /**
  * Object for grouping error messages from (most) exceptions thrown during query execution.
  */
-object SparkCoreErrors {
+private[spark] object SparkCoreErrors {
```

Comment:
cc @karenfeng

Reply:
shall we mark `org.apache.spark.errors` as a private package as well?